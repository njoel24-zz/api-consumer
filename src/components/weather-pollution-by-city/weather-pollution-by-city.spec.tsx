import React from 'react';
import renderer from 'react-test-renderer';

import WeatherAndPollutionByCity from './weather-pollution-by-city.component';

describe('States by country', () => {
	const detail = {
		city: "Milan",
		country: "Italy",
		state: "Lombardia",
		location: {},
		current: {pollution: {}, weather: {}}
	};

	it('renders component and compare with snapshot', () => {
		const component = renderer.create(<WeatherAndPollutionByCity detail={detail}/>);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});