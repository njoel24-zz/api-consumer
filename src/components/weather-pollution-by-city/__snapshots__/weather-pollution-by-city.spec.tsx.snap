// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`States by country renders component and compare with snapshot 1`] = `
<div>
  <div>
    State
    : 
    <strong>
      Lombardia
    </strong>
  </div>
  <div>
    Country
    : 
    <strong>
      Italy
    </strong>
  </div>
  <div>
    City
    : 
    <strong>
      Milan
    </strong>
  </div>
  <div>
    <h4>
      Pollution
    </h4>
    <div>
      aqicn: 
    </div>
    <div>
      aqius: 
    </div>
    <div>
      maincn: 
    </div>
    <div>
      mainus: 
    </div>
  </div>
  <div>
    <h4>
      Weather
    </h4>
    <div>
      hu: 
    </div>
    <div>
      ic: 
    </div>
    <div>
      pr: 
    </div>
    <div>
      tp: 
    </div>
    <div>
      ts: 
    </div>
    <div>
      wd: 
    </div>
    <div>
      ws: 
    </div>
  </div>
</div>
`;
